frontend/src/components/Cases/__tests__/CaseResultSvg.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { expect, test } from "@jest/globals";
+import { create } from "react-test-renderer";
+import CaseResultSvg from "../CaseResultSvg";
+import { CaseResult } from "../../../data/parseReportData";
+import {
+  CheckCircleFill,
+  XCircleFill,
+  ExclamationOctagon,
+} from "react-bootstrap-icons";
+
+test("Case Result Icon Successful Expected True", () => {
```

Comment:
The structure of these tests are identical, so they look like they could be done as some sort of parameterized test where you specify the test case data and expected results in a list of 4 things and then just run the same test over the 4 inputs.
Can you see whether vitest/whatever we're using has something for that, assuming you agree.

Reply:
Yeah, makes sense to me. I don't think there's a built-in way of doing that but I can definitely write a wrapper for it tomorrow if you think it's better - I'm neutral on this.

Reply:
`describe.each` appears to be the way: https://vitest.dev/api/#describe-each

Reply:
Oh, I see, my bad. For the record I think we're using jest, not vitest, but the `describe.each` syntax is the same in both. Thanks!